golang.go

```diff
@@ -0,0 +1,19 @@
+// pacage docummentation
+package main
+
+import "context"
+
+// here is a line coment
+
+// FileGetter is used to retriv the code.
+type FileGetter interface {
+	// GetFiles returs a FilesScanner
```

Comment:
Typo inside comment in word `returs`. Possible corrections:  `return` (78), `returns` (63), `returs` (1)